code/modules/mob/new_player/sprite_accessories.dm

```diff
@@ -49,174 +49,41 @@
 	var/icon_add = 'icons/mob/human_face.dmi'
 	var/flags
 
+	eighties

```

Comment:
Does this change the default hair option from bald?

Reply:
Honestly, no idea. Didn't test for that. Since I assumed that was handled differently.